kubernetes/launch-gtp-eval.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash -eu
+
+HOST_REPO_ROOT=$(git rev-parse --show-toplevel)
+
+DEFAULT_NUM_CPUS=8
+DEFAULT_NUM_GAMES=100
+DEFAULT_NUM_GPUS=1
+DEFAULT_ADV_CONFIG=/go_attack/configs/gtp-amcts.cfg
+DEFAULT_VICTIM_CONFIG=/go_attack/configs/gtp-raw.cfg
+DEFAULT_ADV_MODEL=/shared/victimplay/ttseng-avoid-pass-alive-coldstart-39-20221025-175949/models/t0-s545065216-d136760487/model.bin.gz
+DEFAULT_VICTIM_MODEL=/shared/victims/kata1-b40c256-s11840935168-d2898845681.bin.gz
+
+usage() {
+  echo "Schedules a job that plays a victim (on KataGo-raw) against an A-MCTS"
+  echo "adversary via GTP."
+  echo
+  echo "Results are stored on the go-attack Weka volume by default. On the NAS,"
+  echo "twogtp hits the following error:"
+  echo "  Could not lock file '/shared/gtp-eval/test-20230120-140920/sgfs/game.lock': Input/output error"
```

Comment:
I don't know about "simple" but this error consistently happens if I remove the `USE_WEKA` flag. It also reproduces on `rnn`, e.g., with the following commands
```bash
docker run --gpus '"device=3"' -v ~/go_attack:/go_attack  -v /nas/ucb/ttseng:/nas/ucb/ttseng  -it humancompatibleai/goattack:cpp
# The 1st argument is the output directory. If the output directory
# is the NAS, then we get the error, otherwise twogtp works fine.
/go_attack/kubernetes/gtp-eval.sh /nas/ucb/ttseng/twogtp-debug 1 100 /go_attack/configs/gtp-amcts.cfg /go_attack/configs/gtp-raw.cfg /dev/null /dev/null
```